File: VB/SchedulerDragDropResizeExample/Models/PaymentState.vb
Imports DevExpress.Mvvm

Namespace SchedulerDragDropResizeExample

    Public Class PaymentState
        Inherits BindableBase

        Public Sub New(ByVal id As Integer, ByVal caption As String, ByVal color As String)
            Me.Id = id
            Me.Caption = caption
            Me.Color = color
        End Sub

        Public Property Id As Integer

        Public Property Caption As String

        Public Property Color As String
    End Class
End Namespace

File: CS/SchedulerDragDropResizeExample/Models/PaymentState.cs
using DevExpress.Mvvm;

namespace SchedulerDragDropResizeExample {
    public class PaymentState : BindableBase {
        public PaymentState(int id, string caption, string color) {
            Id = id;
            Caption = caption;
            Color = color;
        }

        public int Id { get; set; }
        public string Caption { get; set; }
        public string Color { get; set; }
    }
}
